With or without this pull request it (develop-2.2) fails to compile, develop compiles successfully. 

```
▷  make                   
==> ibrowse (get-deps)
==> mochiweb (get-deps)
==> webmachine (get-deps)
==> hamcrest (get-deps)
==> riak_pb (get-deps)
==> riakc (get-deps)
==> riak-erlang-http-client (get-deps)
==> ibrowse (compile)
==> mochiweb (compile)
==> webmachine (compile)
==> hamcrest (compile)
==> hamcrest (post_compile)
==> riak_pb (compile)
ERROR: Protobuffs library not present in code path!
ERROR: compile failed while processing /basho/riak-erlang-http-client/deps/riak_pb: rebar_abort
make: *** [all] Error 1
```
I discovered that develop is using a sub-module 'tools' - added it to 'develop-2.2' but still failed with above error 'Protobuffs library not present in code path' 

```
git submodule add --force https://github.com/basho/riak-client-tools.git ./tools 
git submodule update tools
git submodule sync tools
```

@lukebakken -  I know this stuff was reorganized, have you come across this before?

--- a/rebar.config
+++ b/rebar.config
@@ -10,7 +10,7 @@
    {webmachine, "1.10.8", {git, "git://github.com/basho/webmachine", {tag, "1.10.8"}}},
 
    %% riak-erlang-client for riakc_obj
-   {riakc, ".*", {git, "git://github.com/basho/riak-erlang-client", {branch, "develop"}}}
+   {riakc, ".*", {git, "git://github.com/basho/riak-erlang-client", {branch, "develop-2.2"}}}
   ]}.
 {edoc_opts,
  [

